cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
Ensure that the `cinema.urls` module defines a namespace 'cinema' since you are using `namespace="cinema"` in the `include` function.